helmchart/sdp/templates/02.4-deployment-dashboard.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if .Values.dashboard.enabled -}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sdp.fullname" . }}-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "sdp.labelsWithSuffix" (list . "-dashboard") | nindent 4 }}
+  {{- if .Values.dashboard.deployment.annotations }}
+  annotations:
+    {{- tpl (toYaml .Values.deployment.annotations) . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "sdp.selectorLabelsWithSuffix" (list . "-dashboard") | nindent 6 }}
+
+  {{- if .Values.dashboard.deployment.strategy }}
+  strategy:
+    {{- toYaml .Values.dashboard.deployment.strategy | nindent 4 }}
+  {{- end }}
+
+  template:
+    metadata:
+      {{- if .Values.dashboard.deployment.podAnnotations }}
+      annotations:
+        {{- tpl (toYaml .Values.dashboard.deployment.podAnnotations) . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "sdp.selectorLabelsWithSuffix" (list . "-dashboard") | nindent 8 }}
+    spec:
+      {{- if .Values.serviceAccount.name }}
+      serviceAccountName: {{ tpl .Values.serviceAccount.name $ }}
```

Comment:
just question. do we allow templating in the values file?

Reply:
Yes, if you add a template-like strung there, you can apply the template in the file. We do that here, for example:

values.yaml template language:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/338c3c7c79a472557b573271c3665950edd05022/helmchart/sdp/values.yaml#L191-L194

template being parsed at:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/338c3c7c79a472557b573271c3665950edd05022/helmchart/sdp/templates/04.1-ingress-sdp-and-ap.yaml#L27